modules/visual_script/visual_script_nodes.cpp

```diff
@@ -946,39 +946,69 @@ static const char *op_names[] = {
 };
 
 String VisualScriptOperator::get_caption() const {
-	static const char32_t *op_names[] = {
+	switch (op){
```

Comment:
Fix code style to comply with the guidelines (otherwise, static checks will fail on CI):

```suggestion
	switch (op) {
```